src/scicloj/metamorph/core.clj

```diff
@@ -1,17 +1,18 @@
 (ns scicloj.metamorph.core
   (:require [scicloj.metamorph.protocols :as prot]))
 
+(defn ^:deprecated uuid
+  "DEPRECATED: Use clojure.core/random-uuid instead"
+  []
+  (random-uuid))
 
-(defn uuid []
-  (java.util.UUID/randomUUID))
-
-
-(defn check-metamorph-compliant [ctx op]
+(defn check-metamorph-compliant
+  [ctx op]
   (cond
     (keyword? op) ctx
-    (not  (map? ctx)) (throw (IllegalArgumentException.  (str  "Metamorph pipe functions need to return a map, but returned: " ctx "of class: " (type ctx))))
-    (not  (contains? ctx :metamorph/data)) (do (println "Context after operation " op " with meta " (meta op) "does not contain :metamorph/data. This is likely as mistake.") ctx)
-    true ctx))
+    (not (map? ctx)) (throw (IllegalArgumentException.  (str  "Metamorph pipe functions need to return a map, but returned: " ctx "of class: " (type ctx))))
+    (not (contains? ctx :metamorph/data)) (do (println "Context after operation " op " with meta " (meta #'op) "does not contain :metamorph/data. This is likely as mistake.") ctx)
```

Comment:
I am getting a compile error here:

Syntax error compiling var at (scicloj/metamorph/core.clj:14:100).
Unable to resolve var: op in this context

Could you please check ?

Reply:
Interesting, I get that error with `#'op` it looks like I just added the quote as a mistake, let me check!